doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
+when you type `python` in the Windows command prompt the correct Python is
+executed.
+
+Install Microsoft Visual Studio 2017 or 2019 Community Edition (use the
+`build tools`_ from Microsoft). Just like before pick a convenient place to
+clone SciPy. Next to OpenBLAS is often a convenient option (note: not inside
+OpenBLAS folder but next to). Continuing the example from above
 
-1) Install Microsoft Visual Studio 2015 or 2017 Community Edition (use the `build tools`_
-   from Microsoft)
-2) Finally, install Python from https://www.python.org/ (make sure to check the box to install
-   pip)
+    my repo folder
+        ├─── build_openblas.sh
+        ├─── OpenBLAS
+        ├─── SciPy
+                ├─── ...
 
-After you've installed the required software, open an MSYS2 terminal, change to a good
-location to build, and clone SciPy.
+Again using the same generic example folder from above
 
 .. code:: shell
 
-   cd C:\Users\MyUser\Downloads
+   cd C:\Users\Ilhan\Documents\GitHub
    git clone https://github.com/scipy/scipy.git
    cd scipy
    
-Now we need to copy the :code:`openblas.a` file that we've built earlier to the correct
-location. If your Python is installed somewhere like the following:
+Now we need to copy the :code:`openblas.a` file that we've built earlier to the
+correct location. If your Python is installed somewhere like the following:
 
 .. code:: shell
 
-   C:\Users\<user name>\AppData\Local\Programs\Python\Python36\python.exe
+   C:\Users\<user name>\AppData\Local\Programs\Python\Python38\python.exe
 
-
-Then you'll need to put the :code:`openblas.a` file somewhere like the following:
+Then you'll need to put the :code:`openblas.a` file that we previously copied
+and renamed somewhere like the following:
 
 .. code:: shell
 
-   C:\Users\<user name>\AppData\Local\Programs\Python\Python36\Lib
+   C:\Users\<user name>\AppData\Local\Programs\Python\Python38\Lib
+
+Adjust the location accordingly based on where :code:`python.exe` is located.
 
-Adjust the location accordingly based on where :code:`python.exe` is located. Now for a
-sanity check. Type  the following and press enter.
+At this stage, we are done with the OpenBLAS part and hopefully we will not need
+to build OpenBLAS anytime soon. But we tend to build SciPy more often as it is
+on a quicker release cycle. Hence it makes sense to use Windows ``cmd`` or 
+Powershell for the the build as it is a more native tool. This requires placing
+the MinGW compilers on the path. Hence for a sanity check type  the following on
+the command prompt or Powershell
 
 .. code:: shell
 
     gfortran
 
-If you see an error with the above command, :code:`gfortran` is not correctly installed.
-Go back to the "Building OpenBLAS" section and make sure that you have installed the correct
-tools.
+If you see a missing command error with the above, :code:`gfortran` is not
+correctly installed or not on the path. Hence make sure that the following 
```

Comment:
Is there any reason to believe that this folder would already be on the path?
"If you see a missing command error with the above..." suggests that there is some reason it might already be there.

Reply:
Yes by default MSYS2 installation should add it to the path if it didn't then I am remembering incorrectly.

Reply:
It did not on my machine.

Reply:
OK then I'll make it explicit.